lint/lint.py

```diff
@@ -413,8 +413,8 @@ def output_errors_text(errors):
     for error_type, description, path, line_number in errors:
         pos_string = path
         if line_number:
-            pos_string += " %s" % line_number
-        print("%s: %s %s" % (error_type, pos_string, description))
+            pos_string += ":%s" % line_number
+        print("%s: %s (%s)" % (pos_string, description, error_type))
```

Comment:
so this is using `sourcefile:lineno: description (error type)`, so e.g., `2dcontext/drawing-text-to-the-canvas/2d.text.draw.fontface.html:31: setTimeout used; step_timeout should typically be used instead (SET TIMEOUT)`?